package.json

```diff
@@ -10,7 +10,7 @@
     "acceptance": "scripts/acceptance.sh",
     "acceptance-record": "scripts/acceptance_record.sh",
     "assets": "scripts/assets.sh",
-    "bundle-report": "NODE_ENV=production ANALYZE_BUNDLE=true webpack",
+    "bundle-report": "ANALYZE_BUNDLE=true webpack",
```

Comment:
Removed `NODE_ENV` so that we can more easily introspect structure, vs bundle-size. It's easy enough to pass in 
```
$ NODE_ENV=production yarn bundle-report
```